src/converters/piral-blazor/src/interop.ts

```diff
@@ -0,0 +1,163 @@
+export const eventNames = {
+  render: 'render-blazor-extension',
+  navigate: 'navigate-blazor',
+};
+
+const coreLib = 'Piral.Blazor.Core';
+const eventParents: Array<HTMLElement> = [];
+let blazorRootElement: HTMLElement = null;
+
+const globalEventNames = [
+  'abort',
+  'blur',
+  'change',
+  'error',
+  'focus',
+  'load',
+  'loadend',
+  'loadstart',
+  'mouseenter',
+  'mouseleave',
+  'progress',
+  'reset',
+  'scroll',
+  'submit',
+  'unload',
+  'DOMNodeInsertedIntoDocument',
+  'DOMNodeRemovedFromDocument',
+  'click',
+  'dblclick',
+  'mousedown',
+  'mousemove',
+  'mouseup',
+];
+
+function dispatchToRoot(event: Event) {
+  if (!blazorRootElement) blazorRootElement = document.querySelector('#blazor-root');
```

Comment:
I suggest not to have this available on module level.

Instead use `document.getElementById` with `blazor-root` and put `blazor-root` on the module level.

``` ts
const blazorRootId = 'blazor-root';
```

use it here and and in line 44.